scripts/entrypoint.sh

```diff
@@ -14,6 +15,12 @@ echo "Proceeding with JAVA_OPTIONS=$JAVA_OPTIONS"
 
 sed -i.bckp 's#host: .*#host: '$GREMLIN_HOST'#' ${GREMLIN_CONF}
 
+if grep -i 'graph.unique-instance-id=' "$PROPS" 1>/dev/null; then
```

Comment:
@miteshvp I was too fast merging this PR, but the suggestion from @vpavlin is a good one. Definitely worth fixing.